base/setup/reactos/resource.h

```diff
@@ -100,3 +104,50 @@
 #define IDS_PARTITION_TYPE   5201
 #define IDS_PARTITION_SIZE   5202
 #define IDS_PARTITION_STATUS 5203
+
+
+/* Progress Strings */
+#define IDS_FORMATTING_PROGRESS1    5300
+#define IDS_FORMATTING_PROGRESS2    5301
+#define IDS_CHECKING_PROGRESS1      5302
+#define IDS_CHECKING_PROGRESS2      5303
+#define IDS_COPYING                 5304
+#define IDS_MOVING                  5305
+#define IDS_RENAMING                5306
+#define IDS_DELETING                5307
+#define IDS_PREPARE_PARTITIONS      5308
+#define IDS_PREPARE_FILES           5309
+#define IDS_COPYING_FILES           5310
+
+
+/* Error Strings */
+
+// ERROR_CREATE_PARTITION // TODO?
+#define IDS_ERROR_CREATE_PARTITION_TITLE        5405
+#define IDS_ERROR_CREATE_PARTITION              5406
+
+#define IDS_WARN_DELETE_PARTITION_TITLE         5407
+#define IDS_WARN_DELETE_PARTITION               5408
+#define IDS_WARN_DELETE_MBR_EXTENDED_PARTITION  5409
+
+// ERROR_WRITE_PTABLE
+#define IDS_ERROR_WRITE_PTABLE          5410
```

Comment:
indent of 5410 was one space more than 5411 and 5413ff
```suggestion
#define IDS_ERROR_WRITE_PTABLE         5410
```

Reply:
![image](https://github.com/user-attachments/assets/bc205bfa-9630-44b8-bac9-c99f3da8d263)
First line: my original version;
Second line: your suggestion: as you can see with the dots and vertical bar (comparison with 4th line), alignment in your suggestion is _**not**_ 4-space aligned, as was my intent.

Reply:
You are correct. I got fooled by copying out the line into my text editor by a strange habit of github preview. Actually what you used **is** aligned to what is used for 5411, 5413 and ff. Resolved.

Reply:
Indeed, the github web diff has a problem where it visuall shifts one character left or right, probably due to the +/- thing they have in their margin.